src/test/e2e/api/admin/event-search.e2e.test.ts

```diff
@@ -182,13 +246,15 @@ test('should filter events by type', async () => {
 test('should filter events by created by', async () => {
     await eventService.storeEvent({
         type: FEATURE_CREATED,
+        project: 'default',
         createdBy: 'admin1@example.com',
         createdByUserId: TEST_USER_ID + 1,
         ip: '127.0.0.1',
     });
 
     await eventService.storeEvent({
         type: FEATURE_CREATED,
+        project: 'default',
         createdBy: 'admin2@example.com',
```

Comment:
Interesting: did this test not work without the project? If so, why not?

Reply:
Actually since feature created always has project, wanted to make it legit.